src/hu_nmt/data_augmentator/graph_mappers/edge_mapper.py

```diff
@@ -0,0 +1,161 @@
+import networkx as nx
+from collections import defaultdict, Counter
+import nltk
+from typing import List
+
+from hu_nmt.data_augmentator.graph_mappers.graph_similarity_base import GraphSimilarityBase, Edge, Node
+
+
+class EdgeMapper(GraphSimilarityBase):
```

Comment:
Can you please add docstring to this class summarizing what this class uses for similarity?